NASA-AMMOS/aerie-ui
# Scheduling jar upload

Resolves #1436 

To test:

1. Navigate to http://localhost:3000/scheduling/goals/new
2. Fill out the name for the goal
3. Select "JAR File" to display the file input field
4. Select the example procedure file at `/e2e-tests/data/example-procedure.jar`
5. Verify that the code editor is no longer viewable
6. Finish filling out the rest of the goal's fields as desired
7. Click the "Save" button
8. Verify that the goal was successfully created
